examples/async/webpack.config.js

```diff
@@ -4,8 +4,7 @@ var webpack = require('webpack');
 module.exports = {
   devtool: 'cheap-module-eval-source-map',
   entry: [
-    'webpack-dev-server/client?http://localhost:3000',
-    'webpack/hot/only-dev-server',
+    'webpack-hot-middleware/client?path=/__webpack_hmr&timeout=20000',
```

Comment:
[Ah, you're right](https://github.com/glenjamin/webpack-hot-middleware/blob/master/client.js#L4-L9). I'll rip that out.